ansible/configs/kni-osp/software.yml

```diff
@@ -0,0 +1,263 @@
+---
+- name: Step 001 software
+  hosts: utility
+  gather_facts: false
+  become: true
+  tasks:
+    - when: cloud_provider == "osp"
+      block:
+        - name: Update packages
```

Comment:
This can already be done against all the hosts that are provisioned by agnosticd, including this one, if you have `update_packages` set to true.

Reply:
don't want to update all hosts ;)